install/resources/mybb_theme.xml

```diff
@@ -822,6 +822,14 @@ blockquote cite span.highlight {
 	font-size: 13px;
 }
 
+.mycode_me {
```

Comment:
@JordanMussi, parser is used in many places, including plugins, not only these pages. Besides, there's definitely an unfixed bug report about stylesheets not created on upgrade sometimes, so I wouldn't rely on that. PaulBender's solution (style and class at the same time) looks reasonable, but it's been applied only to /me and /slap so far.